ci/validate_asciidoc.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash
 set -uo pipefail
 
+# Install script dependencies
+(set -e; cd rspec-tools && pipenv install)
```

Comment:
IIUC, `set -e` will only terminate the subshell on an error. If you are going for the early failure of the entire script, you'd have to do something like
```suggestion
set -e
cd rspec-tools && pipenv install
set +e
```

Reply:
Indeed, I made a mistake while testing! Thanks for catching this.